### ISSUE TITLE
Sometimes not restoring status after series refresh

### ISSUE THREAD

**Issue by Herjar** [evilhero/mylar#1164]
Most of my issues have 'Archived' as status. Sometimes the status for all issues of a comic is changed to 'Skipped' after a series is refreshed. Not sure why it happens, but I think I found one case it in the log below. It also seems to check for annuals even if I have disabled that?

19-Nov-2015 18:23:16 - DEBUG   :: CP Server Thread-7 : Refreshing comic: [u'17993']
19-Nov-2015 18:23:16 - INFO    :: Thread-30 : Starting update for 1 active comics
19-Nov-2015 18:23:16 - DEBUG   :: Thread-30 : Refreshing :17993
19-Nov-2015 18:23:16 - DEBUG   :: Thread-30 : CV_OneTimer option enabled...
19-Nov-2015 18:23:16 - DEBUG   :: Thread-30 : Gathering the status of all issues for the series.
19-Nov-2015 18:23:16 - DEBUG   :: Thread-30 : Checking how out of whack the series is.
19-Nov-2015 18:23:16 - DEBUG   :: Thread-30 : Deleting all issue data.
19-Nov-2015 18:23:16 - DEBUG   :: Thread-30 : Refreshing the series and pulling in new data using only CV.
19-Nov-2015 18:23:16 - WARNING :: Thread-30 : [DIRECTORY-CHECK] Could not find comic directory: /media/a6a3/home/herjar/data/complete/comics/Invincible (2003)
19-Nov-2015 18:23:16 - INFO    :: Thread-30 : [DIRECTORY-CHECK] Creating comic directory (0777) : /media/a6a3/home/herjar/data/complete/comics/Invincible (2003)
19-Nov-2015 18:23:16 - DEBUG   :: Thread-30 : Successfully changed permissions [0777 / 0660]
19-Nov-2015 18:23:17 - DEBUG   :: Thread-30 : number of issues I counted: 126
19-Nov-2015 18:23:17 - DEBUG   :: Thread-30 : number of issues CV says it has: 125
19-Nov-2015 18:23:17 - DEBUG   :: Thread-30 : vari is set to: yes
19-Nov-2015 18:23:18 - DEBUG   :: Thread-30 : comic[ComicVersion]:noversion
19-Nov-2015 18:23:18 - DEBUG   :: Thread-30 : {'ComicIssues': '126', 'ComicName': u'Invincible', 'ComicVersion': 'noversion', 'FirstIssueID': u'105347', 'ComicPublisher': u'Skybound', 'ComicYear': u'2003', 'ComicImageALT': u'http://static.comicvine.com/uploads/scale_small/6/67663/2185533-01.jpg', 'ComicURL': u'http://www.comicvine.com/invincible/4050-17993/', 'ComicImage': u'http://static.comicvine.com/uploads/scale_large/6/67663/2185533-01.jpg', 'Aliases': 'None'}
19-Nov-2015 18:23:18 - INFO    :: Thread-30 : Now adding/updating: Invincible
19-Nov-2015 18:23:18 - INFO    :: Thread-30 : Sucessfully retrieved details for Invincible
19-Nov-2015 18:23:18 - INFO    :: Thread-30 : Directory (/media/a6a3/home/herjar/data/complete/comics/Invincible (2003)) already exists! Continuing...
19-Nov-2015 18:23:18 - INFO    :: Thread-30 : Image header check: jpeg
19-Nov-2015 18:23:18 - INFO    :: Thread-30 : Successfully retrieved cover for Invincible
19-Nov-2015 18:23:18 - INFO    :: Thread-30 : previous version detected as None - seeing if update required
19-Nov-2015 18:23:20 - DEBUG   :: Thread-30 : there are 126 search results...
19-Nov-2015 18:23:20 - DEBUG   :: Thread-30 : querying range from 0 to 100
19-Nov-2015 18:23:20 - DEBUG   :: Thread-30 : querying range from 100 to 200
19-Nov-2015 18:23:21 - INFO    :: Thread-30 : Sucessfully retrieved issue details for Invincible
19-Nov-2015 18:23:21 - DEBUG   :: Thread-30 : issuedata call references...
19-Nov-2015 18:23:21 - DEBUG   :: Thread-30 : comicid:17993
19-Nov-2015 18:23:21 - DEBUG   :: Thread-30 : comicname:Invincible
19-Nov-2015 18:23:21 - DEBUG   :: Thread-30 : comicissues:126
19-Nov-2015 18:23:21 - DEBUG   :: Thread-30 : calledfrom: None
19-Nov-2015 18:23:21 - DEBUG   :: Thread-30 : issuechk: None
19-Nov-2015 18:23:21 - DEBUG   :: Thread-30 : latestissueinfo: [{'latestiss': u'125', 'latestdate': u'2015-11-30'}]
19-Nov-2015 18:23:21 - DEBUG   :: Thread-30 : issuetype: None
19-Nov-2015 18:23:21 - DEBUG   :: Thread-30 : checking annual db
19-Nov-2015 18:23:21 - DEBUG   :: Thread-30 : [IMPORTER-ANNUAL] - Annual Year:2003
19-Nov-2015 18:23:21 - DEBUG   :: Thread-30 : Default search mode - this will match on ALL search words
19-Nov-2015 18:23:22 - DEBUG   :: Thread-30 : there are 5 search results...
19-Nov-2015 18:23:22 - DEBUG   :: Thread-30 : [IMPORTER-ANNUAL] - there are 5 results.
19-Nov-2015 18:23:22 - DEBUG   :: Thread-30 : description:<br />AnnualÂ of The Invincible Iron Man on-going series.
19-Nov-2015 18:23:22 - DEBUG   :: Thread-30 : description:<p style=""> 2000 <a href="//www.comicvine.com/iron-man-annual/4050-2905/" data-ref-id="4050-2905">Iron Man Annual</a></p>
19-Nov-2015 18:23:22 - DEBUG   :: Thread-30 : description:<p style="">2001 <a href="//www.comicvine.com/iron-man-annual/4050-2905/" data-ref-id="4050-2905">Iron Man Annual</a></p>
19-Nov-2015 18:23:22 - DEBUG   :: Thread-30 : description:<p style="">First Annual for <a href="//www.comicvine.com/iron-man-annual/4050-2905/" data-ref-id="4050-2905">Iron Man Annual</a> after <a href="//www.comicvine.com/iron-man-captain-america-annual-98/4050-30391/" data-ref-id="4050-30391">Iron Man / Captain America Annual '98</a>.</p>
19-Nov-2015 18:23:22 - DEBUG   :: Thread-30 : description:<p style="">1998 <a href="/captain-america/4005-1442/" data-ref-id="4005-1442">Captain America</a> and <a href="/iron-man/4005-1455/" data-ref-id="4005-1455">Iron Man</a> team-up Annual</p>
19-Nov-2015 18:23:22 - DEBUG   :: Thread-30 : Finshed Annual checking.
19-Nov-2015 18:23:22 - INFO    :: Thread-30 : Now adding/updating issues for Invincible
19-Nov-2015 18:23:22 - DEBUG   :: Thread-30 : initiating issue updating - info & status
19-Nov-2015 18:23:50 - INFO    :: Thread-30 : Updating complete for: Invincible
19-Nov-2015 18:23:50 - INFO    :: Thread-30 : imported is : None
19-Nov-2015 18:23:50 - DEBUG   :: Thread-30 : issue: 125 status before chk :Skipped
19-Nov-2015 18:23:50 - INFO    :: Thread-30 : [FILE-RESCAN] Now checking files for Invincible (2003) in /media/a6a3/home/herjar/data/complete/comics/Invincible (2003)
19-Nov-2015 18:23:50 - DEBUG   :: Thread-30 : [FILECHECKER] comic: Invincible
19-Nov-2015 18:23:50 - DEBUG   :: Thread-30 : [FILECHECKER] Looking in: /media/a6a3/home/herjar/data/complete/comics/Invincible (2003)
19-Nov-2015 18:23:50 - DEBUG   :: Thread-30 : there are 0 files.
19-Nov-2015 18:23:50 - DEBUG   :: Thread-30 : [FILECHECKER] you have a total of 0 Invincible comics
19-Nov-2015 18:23:50 - DEBUG   :: Thread-30 : [FILE-RESCAN]comiccnt is:0
19-Nov-2015 18:23:50 - DEBUG   :: Thread-30 : [FILE-RESCAN]multiple_dest_dirs:
19-Nov-2015 18:23:50 - DEBUG   :: Thread-30 : [FILE-RESCAN]dir: /media/a6a3/home/herjar/data/complete/comics/Invincible (2003)
19-Nov-2015 18:23:50 - DEBUG   :: Thread-30 : [FILE-RESCAN]os.path.basename: Invincible (2003)
19-Nov-2015 18:23:50 - INFO    :: Thread-30 : [FILE-RESCAN] Now checking files for Invincible (2003) in :Invincible (2003)
19-Nov-2015 18:23:50 - DEBUG   :: Thread-30 : [FILECHECKER] comic: Invincible
19-Nov-2015 18:23:50 - DEBUG   :: Thread-30 : [FILECHECKER] Looking in: Invincible (2003)
19-Nov-2015 18:23:50 - DEBUG   :: Thread-30 : there are 0 files.
19-Nov-2015 18:23:50 - DEBUG   :: Thread-30 : [FILECHECKER] you have a total of 0 Invincible comics
19-Nov-2015 18:23:50 - DEBUG   :: Thread-30 : [FILE-RESCAN]tmpv filecount: 0
19-Nov-2015 18:23:50 - DEBUG   :: Thread-30 : No issues with identical issue numbering were detected for this series
19-Nov-2015 18:23:50 - DEBUG   :: Thread-30 : mc_issue:[]
19-Nov-2015 18:23:50 - DEBUG   :: Thread-30 : [FILE-RESCAN] IssueID to ignore: ['17993']
19-Nov-2015 18:24:20 - INFO    :: Thread-30 : [FILE-RESCAN] Updated the status of 126 issues for Invincible (2003) that were not found.
19-Nov-2015 18:24:20 - INFO    :: Thread-30 : [FILE-RESCAN] Total files located: 0
19-Nov-2015 18:24:20 - DEBUG   :: Thread-30 : [FILE-RESCAN] arcfiles : 0
19-Nov-2015 18:24:20 - DEBUG   :: Thread-30 : [FILE-RESCAN] havefiles: 0
19-Nov-2015 18:24:20 - DEBUG   :: Thread-30 : [FILE-RESCAN] I have changed the status of 0 issues to a status of Archived, as I now cannot locate them in the series directory.
19-Nov-2015 18:24:20 - DEBUG   :: Thread-30 : [FILE-RESCAN] Ensuring permissions/ownership enforced for series: Invincible
19-Nov-2015 18:24:20 - DEBUG   :: Thread-30 : Successfully changed permissions [0777 / 0660]
19-Nov-2015 18:24:20 - INFO    :: Thread-30 : [FILE-RESCAN] I have physically found 0 issues, ignored 0 issues, snatched 0 issues, and accounted for 0 in an Archived state [ Total Issue Count: 0 / 126 ]
19-Nov-2015 18:24:20 - DEBUG   :: Thread-30 : issue: 125 status after chk :Skipped
19-Nov-2015 18:24:20 - DEBUG   :: Thread-30 : pullupd: None
19-Nov-2015 18:24:20 - DEBUG   :: Thread-30 : lastpubdate: Present
19-Nov-2015 18:24:20 - DEBUG   :: Thread-30 : series_status: Active
19-Nov-2015 18:24:20 - DEBUG   :: Thread-30 : latestissue: #125
19-Nov-2015 18:24:20 - DEBUG   :: Thread-30 : latestissue status: Skipped
19-Nov-2015 18:24:20 - INFO    :: Thread-30 : Checking this week pullist for new issues of Invincible
19-Nov-2015 18:24:20 - INFO    :: Thread-30 : Checking the Weekly Releases list for comics I'm watching...
19-Nov-2015 18:24:20 - DEBUG   :: Thread-30 : This is a one-off for Invincible[ latest issue: 125 ]
19-Nov-2015 18:24:20 - INFO    :: Thread-30 : No issues marked as wanted for Invincible
19-Nov-2015 18:24:20 - INFO    :: Thread-30 : Finished grabbing what I could.
19-Nov-2015 18:24:35 - INFO    :: Thread-30 : Update complete

Here is a log where it keeps the 'Archived' status as expected. Cannot see many differences except mainly:
calledfrom: dbupdate
initiating issue updating - just the info

19-Nov-2015 18:40:20 - DEBUG   :: CP Server Thread-3 : Refreshing comic: [u'17993']
19-Nov-2015 18:40:20 - INFO    :: Thread-12 : Starting update for 1 active comics
19-Nov-2015 18:40:20 - DEBUG   :: Thread-12 : Refreshing :17993
19-Nov-2015 18:40:20 - DEBUG   :: Thread-12 : CV_OneTimer option enabled...
19-Nov-2015 18:40:20 - DEBUG   :: Thread-12 : Gathering the status of all issues for the series.
19-Nov-2015 18:40:20 - DEBUG   :: Thread-12 : Checking how out of whack the series is.
19-Nov-2015 18:40:20 - DEBUG   :: Thread-12 : Deleting all issue data.
19-Nov-2015 18:40:20 - DEBUG   :: Thread-12 : Refreshing the series and pulling in new data using only CV.
19-Nov-2015 18:40:20 - INFO    :: Thread-12 : [DIRECTORY-CHECK] Found comic directory: /media/a6a3/home/herjar/data/complete/comics/Invincible (2003)
19-Nov-2015 18:40:21 - DEBUG   :: Thread-12 : number of issues I counted: 126
19-Nov-2015 18:40:21 - DEBUG   :: Thread-12 : number of issues CV says it has: 125
19-Nov-2015 18:40:21 - DEBUG   :: Thread-12 : vari is set to: yes
19-Nov-2015 18:40:21 - DEBUG   :: Thread-12 : comic[ComicVersion]:noversion
19-Nov-2015 18:40:21 - DEBUG   :: Thread-12 : {'ComicIssues': '126', 'ComicName': u'Invincible', 'ComicVersion': 'noversion', 'FirstIssueID': u'105347', 'ComicPublisher': u'Skybound', 'ComicYear': u'2003', 'ComicImageALT': u'http://static.comicvine.com/uploads/scale_small/6/67663/2185533-01.jpg', 'ComicURL': u'http://www.comicvine.com/invincible/4050-17993/', 'ComicImage': u'http://static.comicvine.com/uploads/scale_large/6/67663/2185533-01.jpg', 'Aliases': 'None'}
19-Nov-2015 18:40:21 - INFO    :: Thread-12 : Now adding/updating: Invincible
19-Nov-2015 18:40:21 - INFO    :: Thread-12 : Sucessfully retrieved details for Invincible
19-Nov-2015 18:40:21 - INFO    :: Thread-12 : Directory (/media/a6a3/home/herjar/data/complete/comics/Invincible (2003)) already exists! Continuing...
19-Nov-2015 18:40:21 - INFO    :: Thread-12 : Image header check: jpeg
19-Nov-2015 18:40:21 - INFO    :: Thread-12 : Successfully retrieved cover for Invincible
19-Nov-2015 18:40:21 - INFO    :: Thread-12 : previous version detected as None - seeing if update required
19-Nov-2015 18:40:23 - DEBUG   :: Thread-12 : there are 126 search results...
19-Nov-2015 18:40:23 - DEBUG   :: Thread-12 : querying range from 0 to 100
19-Nov-2015 18:40:23 - DEBUG   :: Thread-12 : querying range from 100 to 200
19-Nov-2015 18:40:24 - INFO    :: Thread-12 : Sucessfully retrieved issue details for Invincible
19-Nov-2015 18:40:24 - DEBUG   :: Thread-12 : issuedata call references...
19-Nov-2015 18:40:24 - DEBUG   :: Thread-12 : comicid:17993
19-Nov-2015 18:40:24 - DEBUG   :: Thread-12 : comicname:Invincible
19-Nov-2015 18:40:24 - DEBUG   :: Thread-12 : comicissues:126
19-Nov-2015 18:40:24 - DEBUG   :: Thread-12 : calledfrom: dbupdate
19-Nov-2015 18:40:24 - DEBUG   :: Thread-12 : issuechk: None
19-Nov-2015 18:40:24 - DEBUG   :: Thread-12 : latestissueinfo: [{'latestiss': u'125', 'latestdate': u'2015-11-18'}]
19-Nov-2015 18:40:24 - DEBUG   :: Thread-12 : issuetype: None
19-Nov-2015 18:40:24 - DEBUG   :: Thread-12 : checking annual db
19-Nov-2015 18:40:24 - DEBUG   :: Thread-12 : [IMPORTER-ANNUAL] - Annual Year:2003
19-Nov-2015 18:40:24 - DEBUG   :: Thread-12 : Default search mode - this will match on ALL search words
19-Nov-2015 18:40:25 - DEBUG   :: Thread-12 : there are 5 search results...
19-Nov-2015 18:40:25 - DEBUG   :: Thread-12 : [IMPORTER-ANNUAL] - there are 5 results.
19-Nov-2015 18:40:25 - DEBUG   :: Thread-12 : description:<br />AnnualÂ of The Invincible Iron Man on-going series.
19-Nov-2015 18:40:25 - DEBUG   :: Thread-12 : description:<p style=""> 2000 <a href="//www.comicvine.com/iron-man-annual/4050-2905/" data-ref-id="4050-2905">Iron Man Annual</a></p>
19-Nov-2015 18:40:25 - DEBUG   :: Thread-12 : description:<p style="">2001 <a href="//www.comicvine.com/iron-man-annual/4050-2905/" data-ref-id="4050-2905">Iron Man Annual</a></p>
19-Nov-2015 18:40:25 - DEBUG   :: Thread-12 : description:<p style="">First Annual for <a href="//www.comicvine.com/iron-man-annual/4050-2905/" data-ref-id="4050-2905">Iron Man Annual</a> after <a href="//www.comicvine.com/iron-man-captain-america-annual-98/4050-30391/" data-ref-id="4050-30391">Iron Man / Captain America Annual '98</a>.</p>
19-Nov-2015 18:40:25 - DEBUG   :: Thread-12 : description:<p style="">1998 <a href="/captain-america/4005-1442/" data-ref-id="4005-1442">Captain America</a> and <a href="/iron-man/4005-1455/" data-ref-id="4005-1455">Iron Man</a> team-up Annual</p>
19-Nov-2015 18:40:25 - DEBUG   :: Thread-12 : Finshed Annual checking.
19-Nov-2015 18:40:25 - INFO    :: Thread-12 : Now adding/updating issues for Invincible
19-Nov-2015 18:40:25 - DEBUG   :: Thread-12 : initiating issue updating - just the info
19-Nov-2015 18:41:10 - INFO    :: Thread-12 : Updating complete for: Invincible
19-Nov-2015 18:41:10 - INFO    :: Thread-12 : returning to dbupdate module
19-Nov-2015 18:41:10 - DEBUG   :: Thread-12 : Attempting to put the Status' back how they were.
19-Nov-2015 18:41:54 - INFO    :: Thread-12 : In the process of converting the data to CV, I changed the status of 126 issues.
19-Nov-2015 18:41:54 - INFO    :: Thread-12 : I have added 0 new issues for this series that were not present before.
19-Nov-2015 18:41:54 - INFO    :: Thread-12 : [FILE-RESCAN] Now checking files for Invincible (2003) in /media/a6a3/home/herjar/data/complete/comics/Invincible (2003)
19-Nov-2015 18:41:54 - DEBUG   :: Thread-12 : [FILECHECKER] comic: Invincible
19-Nov-2015 18:41:54 - DEBUG   :: Thread-12 : [FILECHECKER] Looking in: /media/a6a3/home/herjar/data/complete/comics/Invincible (2003)
19-Nov-2015 18:41:54 - DEBUG   :: Thread-12 : there are 0 files.
19-Nov-2015 18:41:54 - DEBUG   :: Thread-12 : [FILECHECKER] you have a total of 0 Invincible comics
19-Nov-2015 18:41:54 - DEBUG   :: Thread-12 : [FILE-RESCAN]comiccnt is:0
19-Nov-2015 18:41:54 - DEBUG   :: Thread-12 : [FILE-RESCAN]multiple_dest_dirs:
19-Nov-2015 18:41:54 - DEBUG   :: Thread-12 : [FILE-RESCAN]dir: /media/a6a3/home/herjar/data/complete/comics/Invincible (2003)
19-Nov-2015 18:41:54 - DEBUG   :: Thread-12 : [FILE-RESCAN]os.path.basename: Invincible (2003)
19-Nov-2015 18:41:54 - INFO    :: Thread-12 : [FILE-RESCAN] Now checking files for Invincible (2003) in :Invincible (2003)
19-Nov-2015 18:41:54 - DEBUG   :: Thread-12 : [FILECHECKER] comic: Invincible
19-Nov-2015 18:41:54 - DEBUG   :: Thread-12 : [FILECHECKER] Looking in: Invincible (2003)
19-Nov-2015 18:41:54 - DEBUG   :: Thread-12 : there are 0 files.
19-Nov-2015 18:41:54 - DEBUG   :: Thread-12 : [FILECHECKER] you have a total of 0 Invincible comics
19-Nov-2015 18:41:54 - DEBUG   :: Thread-12 : [FILE-RESCAN]tmpv filecount: 0
19-Nov-2015 18:41:54 - DEBUG   :: Thread-12 : No issues with identical issue numbering were detected for this series
19-Nov-2015 18:41:54 - DEBUG   :: Thread-12 : mc_issue:[]
19-Nov-2015 18:41:54 - DEBUG   :: Thread-12 : [FILE-RESCAN] IssueID to ignore: ['17993']
19-Nov-2015 18:42:31 - INFO    :: Thread-12 : [FILE-RESCAN] Updated the status of 126 issues for Invincible (2003) that were not found.
19-Nov-2015 18:42:31 - INFO    :: Thread-12 : [FILE-RESCAN] Total files located: 0
19-Nov-2015 18:42:31 - DEBUG   :: Thread-12 : [FILE-RESCAN] Adjusting have total to 126 because of this many archive files:126
19-Nov-2015 18:42:31 - DEBUG   :: Thread-12 : [FILE-RESCAN] arcfiles : 126
19-Nov-2015 18:42:31 - DEBUG   :: Thread-12 : [FILE-RESCAN] havefiles: 126
19-Nov-2015 18:42:31 - DEBUG   :: Thread-12 : [FILE-RESCAN] I have changed the status of 0 issues to a status of Archived, as I now cannot locate them in the series directory.
19-Nov-2015 18:42:32 - DEBUG   :: Thread-12 : [FILE-RESCAN] Ensuring permissions/ownership enforced for series: Invincible
19-Nov-2015 18:42:32 - DEBUG   :: Thread-12 : Successfully changed permissions [0777 / 0660]
19-Nov-2015 18:42:32 - INFO    :: Thread-12 : [FILE-RESCAN] I have physically found 0 issues, ignored 0 issues, snatched 0 issues, and accounted for 126 in an Archived state [ Total Issue Count: 126 / 126 ]
19-Nov-2015 18:42:47 - INFO    :: Thread-12 : Update complete

**Comment by evilhero** [comment]
Well the annual checking is a bug, yes. But it doesn't have anything to do with the problems you're experiencing atm.

You can tell from your logs that you posted. It can't locate the series directory for Invincible:
```
19-Nov-2015 18:23:16 - WARNING :: Thread-30 : [DIRECTORY-CHECK] Could not find comic directory: /media/a6a3/home/herjar/data/complete/comics/Invincible (2003)
19-Nov-2015 18:23:16 - INFO :: Thread-30 : [DIRECTORY-CHECK] Creating comic directory (0777) : /media/a6a3/home/herjar/data/complete/comics/Invincible (2003)
```
So it creates the directory, and in doing so the issues that it had on file (probably pointing to the previous file location of the series on your system) cannot be located. Since it can't locate the initial directory, it has to do a complete refresh of the series (which means wiping out any existing issue data that is present). So when it pulls down the issue data, it can't locate any issues in the directory, so they're assigned the default status of Skipped (as if you were adding a new series to your watchlist).

As opposed to the second refresh you posted, where it can now locate the series directory (and thus the files):
```
19-Nov-2015 18:40:21 - INFO :: Thread-12 : Directory (/media/a6a3/home/herjar/data/complete/comics/Invincible (2003)) already exists! Continuing...
```
Because it can locate the directory, and thus the files - it can store the existing status' of said issues prior to refreshing the series. Once the series refresh is finished, it then puts the status' back to how they were previously to ensure that the status' remain as they were prior to the refresh.

From the looks of it, you probably have a great deal of series that are pointing to your old comic location path. Upon a refresh this gets auto-updated to the current Comic Location path that you have specified in your configuration, so it will do this to pretty much every series that has the old comic location path - at least on the first refresh. Subsequent refreshes will be able to store the correct status', etc since the directories will then be created.

There is a way to mass update the locations in the db to the new location, but it requires that the original comic location path be put back prior to running.

**Comment by Herjar** [comment]
I think this is a separate issue actually, because I had this problem before I moved too. And I ran this query on my database to change location for all comics:
UPDATE `comics` SET ComicLocation = REPLACE(ComicLocation '/media/sdq1/home/', '/media/a6a3/home/');
and that seems to have worked fine. Can now refresh all comics. 

Not finding a folder is the usual as I delete the empty folders regularly. The reason why it can find the directory in the second refresh is that an earlier refresh created the directory and I have not deleted it yet. It's still just an empty directory with no files. My issues usually have archived or snatched status, and in the second refresh the archive status is kept after the refresh.

I'll try to find some more examples in the log later and see if I can understand what happens.

**Comment by Herjar** [comment]
This issue is still present for me. It seems to happen sporadically, so hard to track down. 
I updated all the issue status for all my series around Christmas time. 
Currently this has happened to about 45 of 400 series since then, which now show 0/X issues.

This is on my seedbox so the issues are never physically present. I use archived and snatched status.

I found an example here with Walking Dead. I don't have the debug log so not sure exactly what happens, but here are the main events I can find in the log. The full log follows below.
Seems to me like it does not update issue info after downloading the new issue for some reason?

Main events for Walking Dead in log:
29-Feb-2016 11:07	Auto-refresh of series and issue info. Everything ok [ Total Issue Count: 151 / 151 ]
01-Mar-2016 10:59	Weekly check finds new issue of series. Issue not found - status kept as Wanted.
02-Mar-2016 11:38	Auto-refresh of series and issue info. Everything ok [ Total Issue Count: 151 / 152 ]
02-Mar-2016 15:59	RSS check finds new issue 152. Downloads and updates to snatched
04-Mar-2016 11:03	Auto-refresh of series, but not issue info? Is this where the problems occurs?
06-Mar-2016 11:01	Auto-refresh of series and issue info. Old issue status gone, [ Total Issue Count: 0 / 152 ]
06-Mar-2016 16:00	RSS check. Not looking for Walking dead. Already downloaded OK I assume?
06-Mar-2016 16:29	Mylar updated and restarted
06-Mar-2016 17:01	RSS check. Again finds issue 152 and downloads and sets to snatched
08-Mar-2016 16:36	Auto-refresh of series and issue info. Old issue status gone, but updated with the new 1 snatched [ Total Issue Count: 1 / 152 ]

--------------------------

29-Feb-2016 11:06:23 - INFO    :: DBUPDATE : [13/400] Refreshing :The Walking Dead [18166]
29-Feb-2016 11:06:24 - INFO    :: DBUPDATE : [DIRECTORY-CHECK] Found comic directory: /media/a6a3/home/herjar/tmp/comics/The Walking Dead/ (2003)
29-Feb-2016 11:06:27 - INFO    :: DBUPDATE : Now adding/updating: The Walking Dead
29-Feb-2016 11:06:27 - INFO    :: DBUPDATE : Sucessfully retrieved details for The Walking Dead
29-Feb-2016 11:06:27 - INFO    :: DBUPDATE : Directory (/media/a6a3/home/herjar/tmp/comics/The Walking Dead/ (2003)) already exists! Continuing...
29-Feb-2016 11:06:29 - INFO    :: DBUPDATE : Image header check: jpeg
29-Feb-2016 11:06:29 - INFO    :: DBUPDATE : Successfully retrieved cover for The Walking Dead
29-Feb-2016 11:06:29 - INFO    :: DBUPDATE : previous version detected as None - seeing if update required
29-Feb-2016 11:06:35 - INFO    :: DBUPDATE : Sucessfully retrieved issue details for The Walking Dead
29-Feb-2016 11:06:40 - INFO    :: DBUPDATE : Now adding/updating issues for The Walking Dead
29-Feb-2016 11:07:06 - INFO    :: DBUPDATE : Updating complete for: The Walking Dead
29-Feb-2016 11:07:06 - INFO    :: DBUPDATE : returning to dbupdate module
29-Feb-2016 11:07:31 - INFO    :: DBUPDATE : In the process of converting the data to CV, I changed the status of 151 issues.
29-Feb-2016 11:07:31 - INFO    :: DBUPDATE : I have added 0 new issues for this series that were not present before.
29-Feb-2016 11:07:31 - INFO    :: DBUPDATE : [FILE-RESCAN] Now checking files for The Walking Dead (2003) in /media/a6a3/home/herjar/tmp/comics/The Walking Dead/ (2003)
29-Feb-2016 11:07:31 - INFO    :: DBUPDATE : [FILE-RESCAN] Now checking files for The Walking Dead (2003) in : (2003)
29-Feb-2016 11:07:56 - INFO    :: DBUPDATE : [FILE-RESCAN] Updated the status of 151 issues for The Walking Dead (2003) that were not found.
29-Feb-2016 11:07:56 - INFO    :: DBUPDATE : [FILE-RESCAN] Total files located: 0
29-Feb-2016 11:07:56 - INFO    :: DBUPDATE : [FILE-RESCAN] I have physically found 0 issues, ignored 0 issues, snatched 3 issues, and accounted for 148 in an Archived state [ Total Issue Count: 151 / 151 ]

01-Mar-2016 10:59:32 - INFO    :: WEEKLYCHECK : Now adding/updating issues for The Walking Dead

02-Mar-2016 11:37:02 - INFO    :: DBUPDATE : [53/400] Refreshing :The Walking Dead [18166]
02-Mar-2016 11:37:02 - INFO    :: DBUPDATE : [DIRECTORY-CHECK] Found comic directory: /media/a6a3/home/herjar/tmp/comics/The Walking Dead/ (2003)
02-Mar-2016 11:37:07 - INFO    :: DBUPDATE : Now adding/updating: The Walking Dead
02-Mar-2016 11:37:07 - INFO    :: DBUPDATE : Sucessfully retrieved details for The Walking Dead
02-Mar-2016 11:37:07 - INFO    :: DBUPDATE : Directory (/media/a6a3/home/herjar/tmp/comics/The Walking Dead/ (2003)) already exists! Continuing...
02-Mar-2016 11:37:14 - INFO    :: DBUPDATE : Image header check: jpeg
02-Mar-2016 11:37:14 - INFO    :: DBUPDATE : Successfully retrieved cover for The Walking Dead
02-Mar-2016 11:37:14 - INFO    :: DBUPDATE : previous version detected as None - seeing if update required
02-Mar-2016 11:37:19 - INFO    :: DBUPDATE : Sucessfully retrieved issue details for The Walking Dead
02-Mar-2016 11:37:25 - INFO    :: DBUPDATE : Now adding/updating issues for The Walking Dead
02-Mar-2016 11:37:50 - INFO    :: DBUPDATE : Updating complete for: The Walking Dead
02-Mar-2016 11:37:50 - INFO    :: DBUPDATE : returning to dbupdate module
02-Mar-2016 11:38:17 - INFO    :: DBUPDATE : In the process of converting the data to CV, I changed the status of 152 issues.
02-Mar-2016 11:38:17 - INFO    :: DBUPDATE : I have added 0 new issues for this series that were not present before.
02-Mar-2016 11:38:17 - INFO    :: DBUPDATE : [FILE-RESCAN] Now checking files for The Walking Dead (2003) in /media/a6a3/home/herjar/tmp/comics/The Walking Dead/ (2003)
02-Mar-2016 11:38:17 - INFO    :: DBUPDATE : [FILE-RESCAN] Now checking files for The Walking Dead (2003) in : (2003)
02-Mar-2016 11:38:44 - INFO    :: DBUPDATE : [FILE-RESCAN] Updated the status of 152 issues for The Walking Dead (2003) that were not found.
02-Mar-2016 11:38:44 - INFO    :: DBUPDATE : [FILE-RESCAN] Total files located: 0
02-Mar-2016 11:38:44 - INFO    :: DBUPDATE : [FILE-RESCAN] I have physically found 0 issues, ignored 0 issues, snatched 3 issues, and accounted for 148 in an Archived state [ Total Issue Count: 151 / 152 ]


02-Mar-2016 15:59:45 - INFO    :: RSSCHECK : Shhh be very quiet...I'm looking for The Walking Dead issue: 152 (2016) using 32P [RSS]
02-Mar-2016 15:59:46 - INFO    :: RSSCHECK : Could not find Issue 152 of The Walking Dead(2003) using 32P [rss]
02-Mar-2016 15:59:46 - INFO    :: RSSCHECK : Shhh be very quiet...I'm looking for The Walking Dead issue: 152 (2016) using KAT [RSS]
02-Mar-2016 15:59:46 - INFO    :: RSSCHECK : prov  : KAT [RSS][FEABAA49756819C868DA7B4F8E7E297F3F2763B1]
02-Mar-2016 15:59:46 - INFO    :: RSSCHECK : [FAILED_DOWNLOAD_CHECKER] Successfully marked this download as Good for downloadable content
02-Mar-2016 15:59:46 - INFO    :: RSSCHECK : Found The Walking Dead (2016) issue: 152 using KAT [RSS]
02-Mar-2016 15:59:46 - INFO    :: RSSCHECK : matched on The Walking Dead
02-Mar-2016 15:59:46 - INFO    :: RSSCHECK : setting the nzbid for this download grabbed by KAT in the nzblog to : FEABAA49756819C868DA7B4F8E7E297F3F2763B1
02-Mar-2016 15:59:46 - INFO    :: RSSCHECK : [UPDATER] Updating status to snatched
02-Mar-2016 15:59:47 - INFO    :: RSSCHECK : [UPDATER] Updated the status (Snatched) complete for The Walking Dead Issue: 152


04-Mar-2016 11:03:57 - INFO    :: DBUPDATE : [9/400] Refreshing :The Walking Dead [18166]
04-Mar-2016 11:03:57 - INFO    :: DBUPDATE : [DIRECTORY-CHECK] Found comic directory: /media/a6a3/home/herjar/tmp/comics/The Walking Dead/ (2003)
04-Mar-2016 11:04:00 - INFO    :: DBUPDATE : Now adding/updating: The Walking Dead
04-Mar-2016 11:04:00 - INFO    :: DBUPDATE : Sucessfully retrieved details for The Walking Dead
04-Mar-2016 11:04:00 - INFO    :: DBUPDATE : Directory (/media/a6a3/home/herjar/tmp/comics/The Walking Dead/ (2003)) already exists! Continuing...
04-Mar-2016 11:04:03 - INFO    :: DBUPDATE : Image header check: jpeg
04-Mar-2016 11:04:03 - INFO    :: DBUPDATE : Successfully retrieved cover for The Walking Dead
04-Mar-2016 11:04:03 - INFO    :: DBUPDATE : previous version detected as None - seeing if update required
04-Mar-2016 11:04:09 - INFO    :: DBUPDATE : Sucessfully retrieved issue details for The Walking Dead

06-Mar-2016 11:01:51 - INFO    :: DBUPDATE : [5/400] Refreshing :The Walking Dead [18166]
06-Mar-2016 11:01:51 - INFO    :: DBUPDATE : [DIRECTORY-CHECK] Found comic directory: /media/a6a3/home/herjar/tmp/comics/The Walking Dead/ (2003)
06-Mar-2016 11:01:54 - INFO    :: DBUPDATE : Now adding/updating: The Walking Dead
06-Mar-2016 11:01:54 - INFO    :: DBUPDATE : Sucessfully retrieved details for The Walking Dead
06-Mar-2016 11:01:54 - INFO    :: DBUPDATE : Directory (/media/a6a3/home/herjar/tmp/comics/The Walking Dead/ (2003)) already exists! Continuing...
06-Mar-2016 11:01:57 - INFO    :: DBUPDATE : Image header check: jpeg
06-Mar-2016 11:01:57 - INFO    :: DBUPDATE : Successfully retrieved cover for The Walking Dead
06-Mar-2016 11:01:57 - INFO    :: DBUPDATE : previous version detected as None - seeing if update required
06-Mar-2016 11:02:02 - INFO    :: DBUPDATE : Sucessfully retrieved issue details for The Walking Dead
06-Mar-2016 11:02:08 - INFO    :: DBUPDATE : Now adding/updating issues for The Walking Dead
06-Mar-2016 11:02:38 - INFO    :: DBUPDATE : Updating complete for: The Walking Dead
06-Mar-2016 11:02:38 - INFO    :: DBUPDATE : imported is : None
06-Mar-2016 11:02:38 - INFO    :: DBUPDATE : [FILE-RESCAN] Now checking files for The Walking Dead (2003) in /media/a6a3/home/herjar/tmp/comics/The Walking Dead/ (2003)
06-Mar-2016 11:02:38 - INFO    :: DBUPDATE : [FILE-RESCAN] Now checking files for The Walking Dead (2003) in : (2003)
06-Mar-2016 11:03:08 - INFO    :: DBUPDATE : [FILE-RESCAN] Updated the status of 152 issues for The Walking Dead (2003) that were not found.
06-Mar-2016 11:03:08 - INFO    :: DBUPDATE : [FILE-RESCAN] Total files located: 0
06-Mar-2016 11:03:09 - INFO    :: DBUPDATE : [FILE-RESCAN] I have physically found 0 issues, ignored 0 issues, snatched 0 issues, and accounted for 0 in an Archived state [ Total Issue Count: 0 / 152 ]
06-Mar-2016 11:03:09 - INFO    :: DBUPDATE : Checking this week pullist for new issues of The Walking Dead
06-Mar-2016 11:03:09 - INFO    :: DBUPDATE : Checking the Weekly Releases list for comics I'm watching...
06-Mar-2016 11:03:09 - INFO    :: DBUPDATE : No issues marked as wanted for The Walking Dead
06-Mar-2016 11:03:09 - INFO    :: DBUPDATE : Finished grabbing what I could.


06-Mar-2016 16:29:44 - INFO    :: MAIN : Received signal: update
06-Mar-2016 16:29:44 - INFO    :: MAIN : Aborting all threads
06-Mar-2016 16:29:44 - INFO    :: MAIN : Waiting for the DB UPDATE thread to exit
06-Mar-2016 16:29:45 - INFO    :: MAIN : Waiting for the SEARCH thread to exit
06-Mar-2016 16:29:45 - INFO    :: MAIN : Waiting for the RSS CHECK thread to exit
06-Mar-2016 16:29:46 - INFO    :: MAIN : Waiting for the WEEKLY CHECK thread to exit
06-Mar-2016 16:29:47 - INFO    :: MAIN : Waiting for the VERSION CHECK thread to exit
06-Mar-2016 16:29:47 - INFO    :: MAIN : Waiting for the FOLDER MONITOR thread to exit
06-Mar-2016 16:29:52 - INFO    :: MAIN : Mylar is updating...
06-Mar-2016 16:29:55 - INFO    :: MAIN : Mylar is restarting...
06-Mar-2016 16:29:55 - INFO    :: MAIN : Restarting Mylar with ['/usr/bin/python', '/media/a6a3/home/herjar/.mylar/Mylar.py', '-d', '--nolaunch']
.....

06-Mar-2016 17:01:19 - INFO    :: RSSCHECK : Shhh be very quiet...I'm looking for The Walking Dead issue: 152 (2016) using 32P [RSS]
06-Mar-2016 17:01:19 - INFO    :: RSSCHECK : Could not find Issue 152 of The Walking Dead(2003) using 32P [rss]
06-Mar-2016 17:01:19 - INFO    :: RSSCHECK : Shhh be very quiet...I'm looking for The Walking Dead issue: 152 (2016) using KAT [RSS]
06-Mar-2016 17:01:20 - INFO    :: RSSCHECK : prov  : KAT [RSS][FEABAA49756819C868DA7B4F8E7E297F3F2763B1]
06-Mar-2016 17:01:20 - INFO    :: RSSCHECK : [FAILED_DOWNLOAD_CHECKER] Successfully marked this download as Good for downloadable content
06-Mar-2016 17:01:20 - INFO    :: RSSCHECK : Found The Walking Dead (2016) issue: 152 using KAT [RSS]
06-Mar-2016 17:01:20 - INFO    :: RSSCHECK : matched on The Walking Dead
06-Mar-2016 17:01:20 - INFO    :: RSSCHECK : setting the nzbid for this download grabbed by KAT in the nzblog to : FEABAA49756819C868DA7B4F8E7E297F3F2763B1
06-Mar-2016 17:01:20 - INFO    :: RSSCHECK : [UPDATER] Updating status to snatched
06-Mar-2016 17:01:21 - INFO    :: RSSCHECK : [UPDATER] Updated the status (Snatched) complete for The Walking Dead Issue: 152


08-Mar-2016 16:36:06 - INFO    :: DBUPDATE : [7/400] Refreshing :The Walking Dead [18166]
08-Mar-2016 16:36:06 - INFO    :: DBUPDATE : [DIRECTORY-CHECK] Found comic directory: /media/a6a3/home/herjar/tmp/comics/The Walking Dead/ (2003)
08-Mar-2016 16:36:09 - INFO    :: DBUPDATE : Now adding/updating: The Walking Dead
08-Mar-2016 16:36:09 - INFO    :: DBUPDATE : Sucessfully retrieved details for The Walking Dead
08-Mar-2016 16:36:09 - INFO    :: DBUPDATE : Directory (/media/a6a3/home/herjar/tmp/comics/The Walking Dead/ (2003)) already exists! Continuing...
08-Mar-2016 16:36:11 - INFO    :: DBUPDATE : Attempting to retrieve the comic image for series
08-Mar-2016 16:36:12 - INFO    :: DBUPDATE : previous version detected as None - seeing if update required
08-Mar-2016 16:36:18 - INFO    :: DBUPDATE : Sucessfully retrieved issue details for The Walking Dead
08-Mar-2016 16:36:24 - INFO    :: DBUPDATE : Now adding/updating issues for The Walking Dead
08-Mar-2016 16:36:51 - INFO    :: DBUPDATE : Updating complete for: The Walking Dead
08-Mar-2016 16:36:51 - INFO    :: DBUPDATE : returning to dbupdate module
08-Mar-2016 16:37:21 - INFO    :: DBUPDATE : In the process of converting the data to CV, I changed the status of 152 issues.
08-Mar-2016 16:37:21 - INFO    :: DBUPDATE : I have added 0 new issues for this series that were not present before.
08-Mar-2016 16:37:21 - INFO    :: DBUPDATE : [FILE-RESCAN] Now checking files for The Walking Dead (2003) in /media/a6a3/home/herjar/tmp/comics/The Walking Dead/ (2003)
08-Mar-2016 16:37:21 - INFO    :: DBUPDATE : [FILE-RESCAN] Now checking files for The Walking Dead (2003) in : (2003)
08-Mar-2016 16:37:50 - INFO    :: DBUPDATE : [FILE-RESCAN] Updated the status of 152 issues for The Walking Dead (2003) that were not found.
08-Mar-2016 16:37:50 - INFO    :: DBUPDATE : [FILE-RESCAN] Total files located: 0
08-Mar-2016 16:37:50 - INFO    :: DBUPDATE : [FILE-RESCAN] I have physically found 0 issues, ignored 0 issues, snatched 1 issues, and accounted for 0 in an Archived state [ Total Issue Count: 1 / 152 ]

**Comment by Herjar** [comment]
Ok I looked through the log some more and it seems like the DBUPDATE sometimes crashes/stops, and the series it crashes on does not update properly.
Here are the last series updated in my last few DBUPDATES. Some updates go all the way to 400/400, but when it crashes the series ends up with 0/X issues.

I'm assuming other people don't notice this happening because they have the issues on the disk and it finds the issues again when there is a new update?
I use the "Snatched_havetotal = 1" option and archived state. So if the issue status is deleted it does not update status in a later refresh as the issues are not present.

------------------


21-Feb-2016 21:33:18 - INFO    :: DBUPDATE : [400/400] Refreshing :Prophet [18185]
21-Feb-2016 21:33:18 - INFO    :: DBUPDATE : [DIRECTORY-CHECK] Found comic directory: /media/a6a3/home/herjar/tmp/comics/Prophet (1993)
21-Feb-2016 21:33:21 - INFO    :: DBUPDATE : Now adding/updating: Prophet
21-Feb-2016 21:33:21 - INFO    :: DBUPDATE : Sucessfully retrieved details for Prophet
21-Feb-2016 21:33:21 - INFO    :: DBUPDATE : Directory (/media/a6a3/home/herjar/tmp/comics/Prophet (1993)) already exists! Continuing...
21-Feb-2016 21:33:24 - INFO    :: DBUPDATE : Image header check: jpeg
21-Feb-2016 21:33:24 - INFO    :: DBUPDATE : Successfully retrieved cover for Prophet
21-Feb-2016 21:33:24 - INFO    :: DBUPDATE : previous version detected as None - seeing if update required
21-Feb-2016 21:33:27 - INFO    :: DBUPDATE : Sucessfully retrieved issue details for Prophet
21-Feb-2016 21:33:31 - INFO    :: DBUPDATE : Now adding/updating issues for Prophet
21-Feb-2016 21:33:34 - INFO    :: DBUPDATE : Updating complete for: Prophet
21-Feb-2016 21:33:34 - INFO    :: DBUPDATE : returning to dbupdate module
21-Feb-2016 21:33:36 - INFO    :: DBUPDATE : In the process of converting the data to CV, I changed the status of 11 issues.
21-Feb-2016 21:33:36 - INFO    :: DBUPDATE : I have added 0 new issues for this series that were not present before.
21-Feb-2016 21:33:36 - INFO    :: DBUPDATE : [FILE-RESCAN] Now checking files for Prophet (1993) in /media/a6a3/home/herjar/tmp/comics/Prophet (1993)
21-Feb-2016 21:33:36 - INFO    :: DBUPDATE : [FILE-RESCAN] Now checking files for Prophet (1993) in :Prophet (1993)
21-Feb-2016 21:33:39 - INFO    :: DBUPDATE : [FILE-RESCAN] Updated the status of 11 issues for Prophet (1993) that were not found.
21-Feb-2016 21:33:39 - INFO    :: DBUPDATE : [FILE-RESCAN] Total files located: 0
21-Feb-2016 21:33:39 - INFO    :: DBUPDATE : [FILE-RESCAN] I have physically found 0 issues, ignored 0 issues, snatched 0 issues, and accounted for 11 in an Archived state [ Total Issue Count: 11 / 11 ]
21-Feb-2016 21:33:54 - INFO    :: DBUPDATE : Update complete

23-Feb-2016 19:25:44 - INFO    :: DBUPDATE : [214/400] Refreshing :Headspace [72148]
23-Feb-2016 19:25:44 - INFO    :: DBUPDATE : [DIRECTORY-CHECK] Found comic directory: /media/a6a3/home/herjar/tmp/comics/Headspace/ (2014)

25-Feb-2016 21:33:29 - INFO    :: DBUPDATE : [400/400] Refreshing :Prophet [18185]
25-Feb-2016 21:33:29 - INFO    :: DBUPDATE : [DIRECTORY-CHECK] Found comic directory: /media/a6a3/home/herjar/tmp/comics/Prophet (1993)
25-Feb-2016 21:33:32 - INFO    :: DBUPDATE : Now adding/updating: Prophet
25-Feb-2016 21:33:32 - INFO    :: DBUPDATE : Sucessfully retrieved details for Prophet
25-Feb-2016 21:33:32 - INFO    :: DBUPDATE : Directory (/media/a6a3/home/herjar/tmp/comics/Prophet (1993)) already exists! Continuing...
25-Feb-2016 21:33:35 - INFO    :: DBUPDATE : Image header check: jpeg
25-Feb-2016 21:33:35 - INFO    :: DBUPDATE : Successfully retrieved cover for Prophet
25-Feb-2016 21:33:35 - INFO    :: DBUPDATE : previous version detected as None - seeing if update required
25-Feb-2016 21:33:38 - INFO    :: DBUPDATE : Sucessfully retrieved issue details for Prophet
25-Feb-2016 21:33:42 - INFO    :: DBUPDATE : Now adding/updating issues for Prophet
25-Feb-2016 21:33:46 - INFO    :: DBUPDATE : Updating complete for: Prophet
25-Feb-2016 21:33:46 - INFO    :: DBUPDATE : returning to dbupdate module
25-Feb-2016 21:33:50 - INFO    :: DBUPDATE : In the process of converting the data to CV, I changed the status of 11 issues.
25-Feb-2016 21:33:50 - INFO    :: DBUPDATE : I have added 0 new issues for this series that were not present before.
25-Feb-2016 21:33:50 - INFO    :: DBUPDATE : [FILE-RESCAN] Now checking files for Prophet (1993) in /media/a6a3/home/herjar/tmp/comics/Prophet (1993)
25-Feb-2016 21:33:50 - INFO    :: DBUPDATE : [FILE-RESCAN] Now checking files for Prophet (1993) in :Prophet (1993)
25-Feb-2016 21:33:54 - INFO    :: DBUPDATE : [FILE-RESCAN] Updated the status of 11 issues for Prophet (1993) that were not found.
25-Feb-2016 21:33:54 - INFO    :: DBUPDATE : [FILE-RESCAN] Total files located: 0
25-Feb-2016 21:33:54 - INFO    :: DBUPDATE : [FILE-RESCAN] I have physically found 0 issues, ignored 0 issues, snatched 0 issues, and accounted for 11 in an Archived state [ Total Issue Count: 11 / 11 ]
25-Feb-2016 21:34:09 - INFO    :: DBUPDATE : Update complete

29-Feb-2016 15:12:18 - INFO    :: DBUPDATE : [400/400] Refreshing :Prophet [18185]
29-Feb-2016 15:12:18 - INFO    :: DBUPDATE : [DIRECTORY-CHECK] Found comic directory: /media/a6a3/home/herjar/tmp/comics/Prophet (1993)
29-Feb-2016 15:12:21 - INFO    :: DBUPDATE : Now adding/updating: Prophet
29-Feb-2016 15:12:21 - INFO    :: DBUPDATE : Sucessfully retrieved details for Prophet
29-Feb-2016 15:12:21 - INFO    :: DBUPDATE : Directory (/media/a6a3/home/herjar/tmp/comics/Prophet (1993)) already exists! Continuing...
29-Feb-2016 15:12:24 - INFO    :: DBUPDATE : Image header check: jpeg
29-Feb-2016 15:12:24 - INFO    :: DBUPDATE : Successfully retrieved cover for Prophet
29-Feb-2016 15:12:24 - INFO    :: DBUPDATE : previous version detected as None - seeing if update required
29-Feb-2016 15:12:26 - INFO    :: DBUPDATE : Sucessfully retrieved issue details for Prophet
29-Feb-2016 15:12:30 - INFO    :: DBUPDATE : Now adding/updating issues for Prophet
29-Feb-2016 15:12:32 - INFO    :: DBUPDATE : Updating complete for: Prophet
29-Feb-2016 15:12:32 - INFO    :: DBUPDATE : returning to dbupdate module
29-Feb-2016 15:12:34 - INFO    :: DBUPDATE : In the process of converting the data to CV, I changed the status of 11 issues.
29-Feb-2016 15:12:34 - INFO    :: DBUPDATE : I have added 0 new issues for this series that were not present before.
29-Feb-2016 15:12:34 - INFO    :: DBUPDATE : [FILE-RESCAN] Now checking files for Prophet (1993) in /media/a6a3/home/herjar/tmp/comics/Prophet (1993)
29-Feb-2016 15:12:34 - INFO    :: DBUPDATE : [FILE-RESCAN] Now checking files for Prophet (1993) in :Prophet (1993)
29-Feb-2016 15:12:36 - INFO    :: DBUPDATE : [FILE-RESCAN] Updated the status of 11 issues for Prophet (1993) that were not found.
29-Feb-2016 15:12:36 - INFO    :: DBUPDATE : [FILE-RESCAN] Total files located: 0
29-Feb-2016 15:12:36 - INFO    :: DBUPDATE : [FILE-RESCAN] I have physically found 0 issues, ignored 0 issues, snatched 0 issues, and accounted for 11 in an Archived state [ Total Issue Count: 11 / 11 ]
29-Feb-2016 15:12:52 - INFO    :: DBUPDATE : Update complete

02-Mar-2016 12:00:29 - INFO    :: DBUPDATE : [82/400] Refreshing :Tech Jacket [75342]
02-Mar-2016 12:00:29 - INFO    :: DBUPDATE : [DIRECTORY-CHECK] Found comic directory: /media/a6a3/home/herjar/tmp/comics/Tech Jacket/ (2014)
02-Mar-2016 12:00:32 - INFO    :: DBUPDATE : Now adding/updating: Tech Jacket
02-Mar-2016 12:00:32 - INFO    :: DBUPDATE : Sucessfully retrieved details for Tech Jacket
02-Mar-2016 12:00:32 - INFO    :: DBUPDATE : Directory (/media/a6a3/home/herjar/tmp/comics/Tech Jacket/ (2014)) already exists! Continuing...
02-Mar-2016 12:00:34 - INFO    :: DBUPDATE : Image header check: jpeg
02-Mar-2016 12:00:34 - INFO    :: DBUPDATE : Successfully retrieved cover for Tech Jacket
02-Mar-2016 12:00:34 - INFO    :: DBUPDATE : previous version detected as None - seeing if update required
02-Mar-2016 12:00:38 - INFO    :: DBUPDATE : Sucessfully retrieved issue details for Tech Jacket

04-Mar-2016 11:03:57 - INFO    :: DBUPDATE : [9/400] Refreshing :The Walking Dead [18166]
04-Mar-2016 11:03:57 - INFO    :: DBUPDATE : [DIRECTORY-CHECK] Found comic directory: /media/a6a3/home/herjar/tmp/comics/The Walking Dead/ (2003)
04-Mar-2016 11:04:00 - INFO    :: DBUPDATE : Now adding/updating: The Walking Dead
04-Mar-2016 11:04:00 - INFO    :: DBUPDATE : Sucessfully retrieved details for The Walking Dead
04-Mar-2016 11:04:00 - INFO    :: DBUPDATE : Directory (/media/a6a3/home/herjar/tmp/comics/The Walking Dead/ (2003)) already exists! Continuing...
04-Mar-2016 11:04:03 - INFO    :: DBUPDATE : Image header check: jpeg
04-Mar-2016 11:04:03 - INFO    :: DBUPDATE : Successfully retrieved cover for The Walking Dead
04-Mar-2016 11:04:03 - INFO    :: DBUPDATE : previous version detected as None - seeing if update required
04-Mar-2016 11:04:09 - INFO    :: DBUPDATE : Sucessfully retrieved issue details for The Walking Dead

06-Mar-2016 11:07:07 - INFO    :: DBUPDATE : [11/400] Refreshing :Joe Golem: Occult Detective [85767]
06-Mar-2016 11:07:07 - INFO    :: DBUPDATE : [DIRECTORY-CHECK] Found comic directory: /media/a6a3/home/herjar/tmp/comics/Joe Golem Occult Detective/ (2015)
06-Mar-2016 11:07:10 - INFO    :: DBUPDATE : Now adding/updating: Joe Golem: Occult Detective
06-Mar-2016 11:07:10 - INFO    :: DBUPDATE : Sucessfully retrieved details for Joe Golem: Occult Detective
06-Mar-2016 11:07:10 - INFO    :: DBUPDATE : Directory (/media/a6a3/home/herjar/tmp/comics/Joe Golem Occult Detective/ (2015)) already exists! Continuing...
06-Mar-2016 11:07:13 - INFO    :: DBUPDATE : Image header check: jpeg
06-Mar-2016 11:07:13 - INFO    :: DBUPDATE : Successfully retrieved cover for Joe Golem: Occult Detective
06-Mar-2016 11:07:13 - INFO    :: DBUPDATE : previous version detected as None - seeing if update required
06-Mar-2016 11:07:16 - INFO    :: DBUPDATE : Sucessfully retrieved issue details for Joe Golem: Occult Detective

08-Mar-2016 16:44:14 - INFO    :: DBUPDATE : [17/400] Refreshing :Nailbiter [73649]
08-Mar-2016 16:44:14 - INFO    :: DBUPDATE : [DIRECTORY-CHECK] Found comic directory: /media/a6a3/home/herjar/tmp/comics/Nailbiter/ (2014)

10-Mar-2016 18:20:26 - INFO    :: DBUPDATE : [149/400] Refreshing :Transference [83162]
10-Mar-2016 18:20:26 - INFO    :: DBUPDATE : [DIRECTORY-CHECK] Found comic directory: /media/a6a3/home/herjar/tmp/comics/Comics/
10-Mar-2016 18:20:29 - INFO    :: DBUPDATE : Now adding/updating: Transference
10-Mar-2016 18:20:29 - INFO    :: DBUPDATE : Sucessfully retrieved details for Transference
10-Mar-2016 18:20:29 - INFO    :: DBUPDATE : Directory (/media/a6a3/home/herjar/tmp/comics/Comics/) already exists! Continuing...
10-Mar-2016 18:20:31 - INFO    :: DBUPDATE : Attempting to retrieve the comic image for series
10-Mar-2016 18:20:32 - INFO    :: DBUPDATE : previous version detected as None - seeing if update required
10-Mar-2016 18:20:35 - INFO    :: DBUPDATE : Sucessfully retrieved issue details for Transference

12-Mar-2016 17:04:43 - INFO    :: DBUPDATE : [44/400] Refreshing :Black Magick [85558]
12-Mar-2016 17:04:43 - INFO    :: DBUPDATE : [DIRECTORY-CHECK] Found comic directory: /media/a6a3/home/herjar/tmp/comics/Black Magick/ (2015)
12-Mar-2016 17:04:46 - INFO    :: DBUPDATE : Now adding/updating: Black Magick
12-Mar-2016 17:04:46 - INFO    :: DBUPDATE : Sucessfully retrieved details for Black Magick
12-Mar-2016 17:04:46 - INFO    :: DBUPDATE : Directory (/media/a6a3/home/herjar/tmp/comics/Black Magick/ (2015)) already exists! Continuing...
12-Mar-2016 17:04:48 - INFO    :: DBUPDATE : Attempting to retrieve the comic image for series
12-Mar-2016 17:04:49 - INFO    :: DBUPDATE : previous version detected as None - seeing if update required
12-Mar-2016 17:04:52 - INFO    :: DBUPDATE : Sucessfully retrieved issue details for Black Magick

**Comment by Herjar** [comment]
I enabled debug log and ran a manual db update. It crashed with this error. No idea what that means, but seems it's looking for annuals? I have disabled Series-Annual Integration though.

-------------------

14-Mar-2016 14:38:52 - INFO    :: Thread-14 : [50/400] Refreshing :Lumberjanes [72941]
14-Mar-2016 14:38:52 - DEBUG   :: Thread-14 : CV_OneTimer option enabled...
14-Mar-2016 14:38:52 - DEBUG   :: Thread-14 : Gathering the status of all issues for the series.
14-Mar-2016 14:38:52 - DEBUG   :: Thread-14 : Checking how out of whack the series is.
14-Mar-2016 14:38:52 - DEBUG   :: Thread-14 : Deleting all issue data.
14-Mar-2016 14:38:52 - DEBUG   :: Thread-14 : Refreshing the series and pulling in new data using only CV.
14-Mar-2016 14:38:52 - INFO    :: Thread-14 : [DIRECTORY-CHECK] Found comic directory: /media/a6a3/home/herjar/tmp/comics/Lumberjanes/ (2014)
14-Mar-2016 14:38:54 - DEBUG   :: Thread-14 : number of issues I counted: 23
14-Mar-2016 14:38:54 - DEBUG   :: Thread-14 : number of issues CV says it has: 22
14-Mar-2016 14:38:54 - DEBUG   :: Thread-14 : vari is set to: yes
14-Mar-2016 14:38:54 - DEBUG   :: Thread-14 : comic[ComicVersion]:noversion
14-Mar-2016 14:38:54 - DEBUG   :: Thread-14 : {'ComicIssues': '23', 'ComicName': u'Lumberjanes', 'ComicVersion': 'noversion', 'FirstIssueID': u'450114', 'ComicPublisher': u'Boom! Studios', 'ComicYear': u'2014', 'ComicImageALT': u'http://static.comicvine.com/uploads/scale_small/6/67663/3855006-01a.jpg', 'ComicURL': u'http://comicvine.gamespot.com/lumberjanes/4050-72941/', 'ComicImage': u'http://static.comicvine.com/uploads/scale_large/6/67663/3855006-01a.jpg', 'Aliases': 'None'}
14-Mar-2016 14:38:54 - INFO    :: Thread-14 : Now adding/updating: Lumberjanes
14-Mar-2016 14:38:54 - INFO    :: Thread-14 : Sucessfully retrieved details for Lumberjanes
14-Mar-2016 14:38:54 - INFO    :: Thread-14 : Directory (/media/a6a3/home/herjar/tmp/comics/Lumberjanes/ (2014)) already exists! Continuing...
14-Mar-2016 14:38:56 - INFO    :: Thread-14 : Attempting to retrieve the comic image for series
14-Mar-2016 14:38:57 - DEBUG   :: Thread-14 : comic image retrieval status code: 200
14-Mar-2016 14:38:57 - INFO    :: Thread-14 : previous version detected as None - seeing if update required
14-Mar-2016 14:39:01 - DEBUG   :: Thread-14 : there are 23 search results...
14-Mar-2016 14:39:01 - DEBUG   :: Thread-14 : querying range from 0 to 100
14-Mar-2016 14:39:01 - INFO    :: Thread-14 : Sucessfully retrieved issue details for Lumberjanes
14-Mar-2016 14:39:01 - DEBUG   :: Thread-14 : issuedata call references...
14-Mar-2016 14:39:01 - DEBUG   :: Thread-14 : comicid:72941
14-Mar-2016 14:39:01 - DEBUG   :: Thread-14 : comicname:Lumberjanes
14-Mar-2016 14:39:01 - DEBUG   :: Thread-14 : comicissues:23
14-Mar-2016 14:39:01 - DEBUG   :: Thread-14 : calledfrom: dbupdate
14-Mar-2016 14:39:01 - DEBUG   :: Thread-14 : issuechk: None
14-Mar-2016 14:39:01 - DEBUG   :: Thread-14 : latestissueinfo: [{'latestiss': u'23', 'latestdate': u'2016-02-29'}]
14-Mar-2016 14:39:01 - DEBUG   :: Thread-14 : issuetype: None
14-Mar-2016 14:39:01 - DEBUG   :: Thread-14 : checking annual db
14-Mar-2016 14:39:01 - DEBUG   :: Thread-14 : [IMPORTER-ANNUAL] - Annual Year:2014
14-Mar-2016 14:39:01 - DEBUG   :: Thread-14 : Default search mode - this will match on ALL search words
14-Mar-2016 14:40:03 - ERROR   :: Thread-14 : Uncaught exception: Traceback (most recent call last):
  File "/media/a6a3/home/herjar/.mylar/mylar/logger.py", line 158, in new_run
    old_run(*args, **kwargs)
  File "/usr/lib/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/media/a6a3/home/herjar/.mylar/mylar/updater.py", line 158, in dbUpdate
    cchk = mylar.importer.addComictoDB(ComicID, mismatch, calledfrom='dbupdate', annload=annload)
  File "/media/a6a3/home/herjar/.mylar/mylar/importer.py", line 549, in addComictoDB
    issuedata = updateissuedata(comicid, comic['ComicName'], issued, comicIssues, calledfrom, SeriesYear=SeriesYear, latestissueinfo=latestissueinfo)
  File "/media/a6a3/home/herjar/.mylar/mylar/importer.py", line 1195, in updateissuedata
    weeklyissue_check = annual_check(comicname, SeriesYear, comicid, issuetype, issuechk, weeklyissue_check)
  File "/media/a6a3/home/herjar/.mylar/mylar/importer.py", line 1504, in annual_check
    sresults, explicit = mb.findComic(annComicName, mode, issue=None, explicit='all')#,explicit=True)
  File "/media/a6a3/home/herjar/.mylar/mylar/mb.py", line 132, in findComic
    searched = pullsearch(comicapi, comicquery, 0, explicit, type)
  File "/media/a6a3/home/herjar/.mylar/mylar/mb.py", line 79, in pullsearch
    dom = parseString(r.content) #(data)
  File "/usr/lib/python2.7/xml/dom/minidom.py", line 1928, in parseString
    return expatbuilder.parseString(string)
  File "/usr/lib/python2.7/xml/dom/expatbuilder.py", line 940, in parseString
    return builder.parseString(string)
  File "/usr/lib/python2.7/xml/dom/expatbuilder.py", line 223, in parseString
    parser.Parse(string, True)
ExpatError: syntax error: line 1, column 0

**Comment by evilhero** [comment]
That's what I though actually when I read your post again. 

You're hitting  your comicvine api limit - the expaterror gets returned usually when you go over your api limit. 

I just found this out a few days ago, but apparently CV has limited the #of api calls you can do in an hr. If you go over the limit, you get temp banned from the api until the hr mark resets. I believe for volume requests it's 200 api requests / hr.  Of you're updating mylar and you have 400 series that's 400 volume requests, plus 400 issue requests, plus possible annual volume / issue requests. Mylar currently pauses 2s between api requests so you could in theory do 30 api hits in a minute.  Obviously I need to look further into this, as I was only aware of the 1 api /search rule, the max hits/hr is new. 

You can verify this by signing into your api account at: comicvine.gamespot.com/api
It will tell you your hits as soon as you on on the main page.

**Comment by Herjar** [comment]
I ran another update and got the same error on a different series (Skinned). Comicvine API usage currently says:
You have used 53 requests in the last hour for API Path '/volume' (reset in 19 minutes)
You have used 56 requests in the last hour for API Path '/issues' (reset in 19 minutes)
You have used 97 requests in the last hour for API Path '/search' (reset in 19 minutes)
Your request rate is fine

So it does not seem to hit the limit? Maybe the API request fails for some other reason?

**Comment by evilhero** [comment]
Yeah that seems good for api usage. I can see the annuals thing in your logs and yes, it shouldn't be checking if you have it disabled. It might be trying to do a search, but having values of None to search for might be causing some parsing return problems. I'll have to look into this further and see what's going on, or at least try to understand better what's happening. 

Is it happening just when you do the dB update, or do you hit the same error when you do a Refresh Series on one of the series that returned the error?

**Comment by Herjar** [comment]
I tried refreshing the two series which just got errors (Lumberjanes and Skinned) and they refreshed fine. It seems to be a bit random which series it happens on. Maybe if I kept refreshing lots of series individually one would fail eventually?

It does not seem to be something wrong with specific series. Seems different every time which series it crashes on. Which I guess makes it more likely that it is due to some API limit? It says that my request rate is fine though, so a bit confusing.

I could try to enable annuals again and run another update to see if it still fails?

**Comment by evilhero** [comment]
It's either some kind of api throttling that CV is doing but isn't telling anyone they're doing, or that the dbupdate process is calling something different (and trying annuals too) and encountering the error.
